renku/ui/cli/config.py

```diff
@@ -55,33 +69,6 @@
    :description: Unset entry <key> renku config.
    :extended:
 
-Query values
-~~~~~~~~~~~~
-
-You can display all configuration values with:
-
-.. code-block:: console
-
-    $ renku config show
-    [renku "interactive"]
-    default_url = /lab
-
-Both local and global configuration files are read. Values in local
-configuration take precedence over global values. Use ``--local`` or
-``--global`` flag to read corresponding configuration only.
-You can provide a KEY to display only its value:
-
-.. code-block:: console
-
-    $ renku config show interactive.default_url
-    default_url = /lab
-
-.. cheatsheet::
```

Comment:
Did you intend to remove the cheatsheet entry?

Reply:
Ouch! Nope, that was not intended. I'll restore it